decisions/log/0034-common-urn-structure-productIDs.md

```diff
@@ -0,0 +1,214 @@
+ - short and memorable, ideally reflecting the name or purpose of the organization or entity
+
+Typically, namespace identifiers either refer to the organization which defines the given namespace (such as “ieee”, “iso”), or refers to the identifier itself (“epc”, "uuid”, “isbn”, etc.)
+
+After having gathered feedback on two options:
+- **pfi**
+  pfi could stand for “PACT Footprint Identification” or “Product Footprint Identification” (depending on community preference). The namespace would include all identifiers which facilitate the exchange of product (sustainability) footprint information
+- **pact**
+  PACT stands for “Partnership for Carbon Transparency” and would be a namespace to indicate all identifiers relevant for the industry-agnostic exchange of product carbon footprint information.
+
+After consultation with the PACT community, no strong opinions formed in favor or against the above two options, although a general preference formed in favor of **pact**. We recommend therefore to proceed with **pact** as the recommended namespace option. This is our recommendation given **pact** is a term our community is already familiar with, whereas **pfi** introduces a new abbreviation which could cause more confusion.
+
+
+### Part 2: Examples
+
+The below examples are proposed for inclusion in the PACT Tech Specs
+Product ID Examples
+
+#### Ex 1: Custom Product Ids (Product Codes)
+This example would replace https://wbcsd.github.io/data-exchange-protocol/v2/#dt-productid-custom
+
+    urn:pact:buyerfqdn.com:buyer-id:$CUSTOM-PRODUCT-CODE$
+    
+    urn:pact:buyer.com:buyer-id:1234
+>
+
+    urn:pact:buyerfqdn.com:supplier-id:$CUSTOM-PRODUCT-CODE$
+    
+    urn:pact:supplier.com:supplier-id:1234
+
+#### Ex 2: Product ID defined by TfS
+TfS platform (i.e. SiGREEN) creates UUID credentials for the PCF. With the next release, there will 
+be a UUID for any entity.
+
+    urn:pfi:sigreen.siemens.com: component-identifier:4303F98A-785F-42CD-8DE0-0B79F34FCD40
+
+#### Ex3: Product ID defined by Catena-X
+Per the Catena-X specification, productID corresponds with Industry Core manufacturerPartId. 
+See full details on the [PCF Specification](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.pcf/7.0.0/Pcf.ttl) and
+[PartTypeInformation](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.part_type_information/1.0.0/PartTypeInformation.ttl)
+
+Per Catena-X specification, manufacturerPartID is defined as the “Part ID as assigned by 
+the manufacturer of the part. The part ID identifies the part in the manufacturer`s dataspace. 
+The part ID references a specific version of a part. The version number must be included in 
+the part ID if it is available. The part ID does not reference a specific instance of a part 
+and must not be confused with the serial number."
+
+This specification aligns with the PACT proposal. For example 
+
+    urn:gtin:4712345060507
```

Comment:
Why is a GTIN example shown in the Catena-X chapter? 

I suppose a more fitting example uses the Catena-X UUID format. Please clarify with Catena-X e.V.
 >>urn:uuid:580d3adf-1981-44a0-a214-13d6ceed9379

Reply:
Thanks for the feedback! This proposal was indeed discussed with individuals who are also involved in the Catena-X initiative, but we always welcome others to share feedback as well! This example in particulate is taken directly from the Catena-X 7.0.0 specification, where an example value of a `gtin` urn is shown. (https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.pcf/7.0.0/Pcf.ttl#L133) That said, indeed in the PACT context we expect GTIN identifiers to be relevant across many sectors, and we will include the gtin example in a section not specifically referring to Catena-X.